app/src/main/java/de/westnordost/streetcomplete/quests/diet_type/AddVegetarian.java

```diff
@@ -0,0 +1,43 @@
+package de.westnordost.streetcomplete.quests.diet_type;
+
+import android.os.Bundle;
+
+import javax.inject.Inject;
+
+import de.westnordost.streetcomplete.R;
+import de.westnordost.streetcomplete.data.osm.SimpleOverpassQuestType;
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder;
+import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataDao;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+import de.westnordost.streetcomplete.quests.YesNoQuestAnswerFragment;
+
+public class AddVegetarian extends SimpleOverpassQuestType
+{
+	@Inject public AddVegetarian(OverpassMapDataDao overpassServer) { super(overpassServer); }
+
+	@Override protected String getTagFilters()
+	{
+		return "nodes, ways with ( amenity ~ restaurant|cafe|pub|fast_food|bar ) and !diet:vegetarian";
```

Comment:
I need to ask here: Why bars and pubs? These are not primarily eating places. Especially bars not. (False friends for Germans. `amenity=bar` is not a German "Bar", that's a `amenity=pub`). `amenity=bar` is something like a coctail bar.